talk/talk-api.yaml

```
@@ -1868,6 +1868,29 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ResponseError'
+  /services/transcribe/{id}:
```

```suggestion
  /services/transcribe:
```
I would make this endpoint with `requestBody` `audio/*`, which will be stored in OBS, transcribed, deleted and returned

and use /recordings/{id}/transcribe instead of this endpoint